specs/release.md

```diff
@@ -24,147 +24,182 @@ Direct commits to `main` are PROHIBITED except for emergency hotfixes when the p
+- The hotfix PR remains open throughout the incident
+
+#### Parallel Work During Incident
+
+The hotfix branch serves as the trunk during the incident. To collaborate or work on related fixes in parallel:
+
+1. Create a feature branch from the hotfix branch (not from `main`)
+2. Implement changes and test locally
+3. Create a PR from feature branch to the hotfix branch (not to `main`)
+4. Merge feature branch into hotfix branch
+5. Hotfix branch deployment pipeline runs automatically with the merged changes
+
+This allows multiple developers or workstreams to contribute fixes that are tested together before production deployment.
+
+#### Development Environment Blocking
+
+To prevent confusion during incidents, the normal development deployment workflow SHOULD be modified to block deployments when a hotfix PR is open:
+
+**Blocking behavior:**
+- Check for open PRs with the `hotfix` label before deploying to development
+- If a hotfix PR exists: Skip development deployment, post comment explaining dev is in "hotfix mode"
+- Regular development work can still merge to `main` during the incident
+- Development environment remains at the hotfix branch state until incident is resolved
+
+**Rationale:** This ensures the development environment accurately reflects the production hotfix state and prevents unrelated `main` changes from overwriting the hotfix deployment during testing.
+
+#### Resolution and Merge
+
+When the incident is fully resolved and production is stable:
+
+**1. Manual Merge**
+- Manually merge the hotfix PR to `main` (do not auto-merge)
+- Use squash merge or regular merge based on commit history clarity preferences
+- Ensure PR has final incident summary and deployed version tags documented
+
+**2. Automatic Development Deployment**
+- Merging the hotfix PR to `main` automatically triggers the normal CI/CD pipeline
+- This deploys `main` (now including all hotfix changes) to development
+- Development environment is restored to track `main` instead of the hotfix branch
+- The merge removes the blocking condition, allowing normal dev deployments to resume
+
+**3. Tag Cleanup (Optional)**
+- Multiple hotfix version tags may exist (e.g., `v25.1.1`, `v25.1.2`, `v25.1.3`)
+- All tags remain in git history as a record of production deployments during the incident
+- The final tag represents the version that resolved the incident
+
+**4. Post-Incident Documentation**
+- Add incident summary to the merged PR as a comment (optional but recommended)
+- Link to any post-mortem documents or incident reports
+- Document what was learned and any follow-up work needed
+
+#### Workflow Requirements
+
+The automated portions of the hotfix workflow MUST satisfy these requirements:
+
+**Hotfix branch creation:**
+- Runs on manual trigger (GitHub Actions workflow dispatch)
+- Creates branch from actual production commit (via Deployments API)
+- Generates unique, timestamped branch names
+- Auto-increments version numbers correctly
```

Comment:
No need to. Just need to increment when we tag after a successful deploy. The tag will now be the only place we keep the version.